test/dice_test.coffee

```diff
@@ -0,0 +1,27 @@
+Helper = require('../node_modules/hubot-test-helper/src/index.coffee')
+
+helper = new Helper('../scripts/dice.coffee')
+
+expect = require('chai').expect
```

Comment:
I cannot, yet. This is what hubot-test-helper did. I can drop it, and look into asserting the Array's are equal.